simple_salesforce/api.py

```diff
@@ -800,20 +800,27 @@ def describe_layout(self, record_id, headers=None):
             )
         return self.parse_result_to_json(result)
 
-    def get(self, record_id, headers=None):
+    def get(self, record_id, headers=None, fields=None):
```

Comment:
I think we can actually just refactor this to pass **kwargs in def get as well as in the result = self._call_salesforce and pass **kwargs there as well. Then the user can include any potential additional parameters in the future and pass fields such that params={'fields':'field1,field2,field3,etc'}